pkg/operation/botanist/component/vpnseedserver/vpn_seed_server.go

```diff
@@ -69,8 +69,9 @@ const (
 	OpenVPNPort = 1194
 	// HighAvailabilityReplicaCount is the replica count used when highly available VPN is configured.
 	HighAvailabilityReplicaCount = 2
-	metricsPort                  = 15000
 	metricsPortName              = "metrics"
+	// MetricsPort is the port metrics can be scraped at.
+	MetricsPort = 15000
```

Comment:
Why is this public now?

Reply:
It's used for the one-time migration in `gardenlet/app/app.go`.
`